XenModel/FriendlyNames.resx

```diff
@@ -1841,4 +1841,58 @@
   <data name="Label-host.edition-desktop-cloud" xml:space="preserve">
     <value>XenApp/XenDesktop [Citrix] Cloud</value>
   </data>
+  <data name="Label-host.edition-xcp-ng" xml:space="preserve">
+    <value>XCP-NG</value>

```

Comment:
Thank you for the PR.

The xenserver/xenadmin repo is currently focused on supporting Citrix XenServer. This means that there is no effort in place for other use-cases, to ensure applicability and avoid regressions going forward. Hence at this point, we don't wish to accept this change.

That said, note that XenCenter is licensed under BSD2 in order to make it possible for people to potentially fork the application for other use-cases, which are not covered here.